File: src/main/java-graphql/io/github/graphql/resolver/OrganizationIdentityProviderResolver.java
package io.github.graphql.resolver;

import io.github.graphql.model.*;

/**
 * Resolver for OrganizationIdentityProvider
 */
@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:48+0800"
)
public interface OrganizationIdentityProviderResolver {

    @javax.validation.constraints.NotNull
    ExternalIdentityConnectionTO externalIdentities(OrganizationIdentityProviderTO organizationIdentityProviderTO, String after, String before, Integer first, Integer last) throws Exception;

}

File: src/main/kotlin-graphql/io/github/graphql/resolver/OrganizationIdentityProviderResolver.kt
package io.github.graphql.resolver

import io.github.graphql.model.*

/**
 * Resolver for OrganizationIdentityProvider
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-06T19:48:04+0800"
)
interface OrganizationIdentityProviderResolver {

    @Throws(Exception::class)
    fun externalIdentities(organizationIdentityProviderTO: OrganizationIdentityProviderTO, after: String?, before: String?, first: Int?, last: Int?): ExternalIdentityConnectionTO

}
